app/components/dynamictable/dynamictable.scss

```diff
@@ -117,6 +128,28 @@
           }
         }
       }
+      td:nth-child(1) {
+        //max-width: 200px;
+        //padding-right: 15%;
+        //width: 10%;
+      }
+      td:nth-child(2) {
+        //max-width: 300px;
```

Comment:
Can you remove commented out code for me when you are able?

Reply:
Reminder to remove the commented out code ^

Reply:
@pcast11 Do you think you'll be able to return to this pull request and resolve the feedback?

Reply:
Hi @castastrophe ! Of course, I would be happy to do that. I had been waiting to see Brian Proffitt's answer to your question to whether the feature was needed or not, but I can't find that on here anymore. I will get to it soon, thanks!

Reply:
Oh I chatted with him & Ellie directly and they love it! 👍